### DISCUSSION TITLE
monitor multiple storage /hard disks

### DISCUSSION THREAD

**Discussion by t0mZ4** [syagent/agent-2#24] [Ideas]
hi
in many cases, we have multiple SSD on our servers. Currently sysagent is monitoring both in Total Storage, but showing only one in the detail view.

also we need to have an alert if either of the SSDs is reaching full capacity, because total storage is not giving accurate result.

![image](https://user-images.githubusercontent.com/31729091/222951758-71c4084e-a0a6-4d4b-ad91-ee7792cff09f.png)

**Comment by Rashe** [comment]
Hi, got it. It's a bug, not a feature :) I will create an issue.

**Comment by Rashe** [comment]
Hey Tom, the web view of disks is fixed. 

Can you please check on your end? I have only one server with several disks.

**Comment by t0mZ4** [comment]
looks good here, how would the alarm work?  based on the total or, what I prefer, if each disk reaches 80% and 90% and alarm will trigger.

thanks for your work on this!
![image](https://user-images.githubusercontent.com/31729091/230738724-795dd84c-af61-4ea9-bbf0-71dfcf6a4aa8.png)

**Reply by Rashe** [reply]
Each disk will have separate alarm. Btw, I check only disks with 300mb and more.

**Comment by Rashe** [comment]
And thanks for pointing out, there was a bug with disks for two years :)

**Comment by t0mZ4** [comment]
perfect, i'll close this then, very useful fix!
